WordPress/Tests/VIP/SlowDBQueryUnitTest.inc

```diff
@@ -28,10 +28,10 @@ $test = array(
 
 	// Single-line statements.
 	'tax_query' => array(), // Bad.
-	'tax_query' => array(), // WPCS: tax_query ok.
+	'tax_query' => array(), // WPCS: slow query ok.
 
 	// Multi-line statement.
-	'tax_query' => array( // WPCS: tax_query ok.
+	'tax_query' => array( // WPCS: slow query ok.
 		array(
 			'taxonomy' => 'foo',
```

Comment:
I'd like to suggest having at least one example with the `tax_query` whitelist comment so that the fact that it's still respected is covered by the tests.
In case a deprecation warning would be thrown for it, this would still result in a warning on that line (in contrast to `slow_query` which would have no warning).